src/math_util.rb

```diff
@@ -0,0 +1,43 @@
+class MathUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers&.select(&:even?)&.sum
+  end
+
+  def self.two_largest_numbers(numbers)
+    return nil if numbers.nil? ||  numbers.size < 2
+
+    firstNumber = numbers[0]
+    secondNumber = numbers[1]
+
+    if firstNumber < secondNumber
+      firstNumber, secondNumber = secondNumber, firstNumber
+    end
+
+    numbers[2...numbers.size].each { |x|
+      if x > firstNumber
+        secondNumber = firstNumber
+        firstNumber = x
+      elsif x > secondNumber
+        secondNumber = x
+      end
+    }
+
+    [firstNumber, secondNumber]
+  end
+
+  def self.exists_duplicates(numbers)
+    return nil if numbers.nil? || numbers.size < 2
```

Comment:
Given that you have to find if there are duplicates in an array or not, how does it help you that you return `nil`? Ask yourself how would you use a function like this? How can it help you in how you'll use it if it returns `nil`?

Reply:
I can raise an exception or return ```false```. But I think the best solution is to return ```false```.
What do you think?